crates/bevy_ecs/src/entity/map_entities.rs

```diff
@@ -18,7 +19,7 @@ use bevy_utils::EntityHashMap;
 ///
 /// ```
 /// use bevy_ecs::prelude::*;
-/// use bevy_ecs::entity::{EntityMapper, MapEntities};
+/// use bevy_ecs::entity::{SceneEntityMapper, MapEntities, EntityMapper};
```

Comment:
It's in prelude now:
```suggestion
/// use bevy_ecs::entity::{SceneEntityMapper, MapEntities};
```